### ISSUE TITLE
How about the reverse (generate script by recording actions)?

### ISSUE THREAD

**Issue by ogencoglu** [tebelorg/TagUI-Python#2]
This is not an issue per se but more  of a question: Is there a way to automatically generate the commands for a series of actions as a script, out of the actions itself when performed once. 

For example, a script records the actions and one can simply perform the actions once, then the corresponding commands are created.

**Comment by kensoh** [comment]
Hi @ogencoglu thanks for raising this question. This project is based on TagUI, another project which I created that converts natural-language-looking instructions into JS code to automate workflows.

For that project, I made a Chrome extension that records user actions in Chrome and outputs the script of actions to be done. Technically the same thing can be done here.

But the problem is recording typically captures identifiers with significant level of error, making it time-consuming to check and modify later. Thus for myself, I do it manually instead of using recorder. And for this project, there is no plan to make a recorder to generate scripts of the workflow actions.

**References to recorder code**
https://github.com/kelaberetiv/TagUI/blob/master/src/chrome/recorder.js#L379
https://github.com/kelaberetiv/TagUI/blob/master/src/chrome/tagui.js#L281
